gui/docks/roidock.cpp

```diff
@@ -69,14 +69,24 @@ void RoiDock::initUi()
 	connect(roiView, SIGNAL(newSizeHint(QSize)),
 			view, SLOT(updateSizeHint(QSize)));
 
-	connect(uibtn->roiButtons, SIGNAL(clicked(QAbstractButton*)),
-			 this, SLOT(processRoiButtonsClicked(QAbstractButton*)));
 	connect(roiView, SIGNAL(newSelection(const QRect&)),
 			this, SLOT(processNewSelection(const QRect&)));
 	connect(bandsSlider, SIGNAL(valueChanged(int)),
 			this, SLOT(processBandsSliderChange(int)));
 	connect(bandsSlider, SIGNAL(sliderMoved(int)),
 			this, SLOT(processBandsSliderChange(int)));
+
```

Comment:
Didn't we have icons for reset and apply in your last version? Use "dialog-apply" for Apply and "undo" for Revert from Theme. Should also be in menu.